pyvlx/opening_device.py

```diff
@@ -283,6 +283,51 @@ class RollerShutter(OpeningDevice):
     """RollerShutter object."""
 
 
+class DualRollerShutter(RollerShutter):
+    """Dual RollerShutter object."""
+
+    async def set_position(self, position, wait_for_completion=True, curtain="both"):
+        """Set dual roller shutter to the desired position.
```

Comment:
The problem here is, that this definition conflichts with the base implementation. I think the way around it could be to add a **kwargs to the base class (not a big fan of this approach, bc it makes refactoring hard).

Another idea would be to add an additional method to the DualRollerShutter.